selectrum.el

```diff
@@ -873,55 +873,43 @@ PREDICATE, see `read-buffer'."
   "Read file name using Selectrum. Can be used as `read-file-name-function'.
 For PROMPT, DIR, DEFAULT-FILENAME, MUSTMATCH, INITIAL, and
 PREDICATE, see `read-file-name'."
-  (let* ((minibuffer-completing-file-name t)
-         (dir (file-name-as-directory
-               (expand-file-name (or dir default-directory))))
-         (candidates
-          (lambda (input)
-            (let* ((new-input (file-name-nondirectory input))
-                   (dir (or (file-name-directory input) dir))
-                   (entries (selectrum--map-destructive
-                             (lambda (cell)
-                               (let* ((name (car cell))
-                                      (type (nth 0 (cdr cell)))
-                                      ;; Check if directory (fast) or
-                                      ;; symlink to directory
-                                      ;; (slower).
-                                      (isdir (or (eq t type)
-                                                 (and (stringp type)
-                                                      (file-directory-p
-                                                       (concat dir name))))))
-                                 (propertize
-                                  name
-                                  'selectrum-candidate-display-suffix
-                                  (when isdir "/")
-                                  'selectrum-candidate-full
-                                  (concat dir name (when isdir "/")))))
-                             (cl-delete-if
-                              (lambda (cell)
-                                (and predicate
-                                     (not (funcall
-                                           predicate
-                                           (concat dir (car cell))))))
-                              (condition-case _
-                                  (directory-files-and-attributes
-                                   dir nil "^[^.]\\|^.[^.]" 'nosort)
-                                (file-error)
-                                ;; May happen in case user quits out
-                                ;; of a TRAMP prompt.
-                                (quit))))))
```

Comment:
The new code doesn't handle this case. You get a `post-command-hook` error.

Reply:
Sorry, fixed by c3e5e9a2fe9f54ae05f21223e520c72adf385705